### ISSUE TITLE
Feature: mark as unread

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#468]
In some instances it may be useful to mark chats as unread to catch the attention of the host phone owner.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.9.38](https://github.com/open-wa/wa-automate-nodejs/compare/1.9.37...1.9.38) (2020-05-22)

- Release 1.9.38 [`1757280`](https://github.com/open-wa/wa-automate-nodejs/commit/17572802a937dac5b2510cdbeb2f7e63fac69af3)
- feature: implemented `markAsUnread` GA #468 [`3ed98bb`](https://github.com/open-wa/wa-automate-nodejs/commit/3ed98bbc96d597aec8392a4376722b81c47a4406)
